installer/manifests/keptn/charts/control-plane/templates/rbac.yaml

```diff
@@ -192,6 +192,24 @@ rules:
       - update
       - get
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: keptn-get-webhook-config
+  labels:
+    {{ include "control-plane.labels" . | nindent 4 }}
+    app.kubernetes.io/name: keptn-get-webhook-config
+    app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
+    app.kubernetes.io/component: {{ include "control-plane.name" . }}
+rules:
+  - apiGroups:
+      - keptn-webhook-config
```

Comment:
the name of the configMap should rather be listed in the `resourceNames` property - see https://github.com/keptn/keptn/blob/f4d5242055f9fd3aa398a4b6bef8ae4b1f902893/installer/manifests/keptn/charts/control-plane/templates/rbac.yaml#L232 for an example :)